digid_eherkenning/models/certificates.py

```diff
@@ -147,9 +162,24 @@ class Meta:
                 violation_error_message=_(
                     "This configuration and certificate combination already exists."
                 ),
-            ),
+            )
         ]
 
+    def clean(self):
+        if not (
+            self.certificate.valid_from
+            <= self.activation_date
+            <= self.certificate.expiry_date
+        ):
+            raise ValidationError(
+                "The activation date must fall within the period defined by 'valid_from' and 'expiry_date'."
+            )
+        return super().clean()
+
+    def save(self, **kwargs):
+        self.clean()
+        return super().save(**kwargs)
```

Comment:
Please revert this - Django deliberately doesn't do this.

The `clean` method is called as part of form validation (in the admin and model forms)